src/renderer/components/+catalog/detail-item-registrations.injectable.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import { computed } from "mobx";
+import rendererExtensionsInjectable from "../../../extensions/renderer-extensions.injectable";
+import coreDetailItemRegistrationsInjectable from "./core-detail-item-registrations.injectable";
+import { orderByPriority } from "../../utils/order-by-priority";
+
+const detailItemRegistrationsInjectable = getInjectable({
```

Comment:
When first time see this code/file it's hard to predict what kind of type it returns/accepts? (without extra navigating in codebase).
Maybe we should start using definition with type? Kinda `const detailItemRegistrationsInjectable: Injectable<string[]> = ...` so `detailItemRegistrations` expected to be strings array. Wdyt?

Reply:
Hm. I think it would help a lot if I just extracted implementation to separate function.